deployment-template/k8s/README.md

```diff
@@ -0,0 +1,29 @@
+# Deploying with Kubernetes (k8s)
+This directory contains a Kubernetes configuration file capable of provisioning a Data Prepper cluster.
+
+Use [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) to apply these configurations files with `kubectl apply -f data-prepper.yaml` 
+
+This will provision a scalable Data Prepper cluster capable of receiving trace data, however it is up to the user to decide how the service will be accessed by clients. A few examples include:
+1. Adding a load balancer with an ingress controller (for server-side load balancing)
+2. Exposing Kubernetes DNS so that clients can retrieve the addresses of Data Prepper pods (for client-side load balancing)
+3. Using something like [ExternalDNS](https://github.com/kubernetes-sigs/external-dns) to synchronize an external DNS (for client-side load balancing)
+
+---
+
+## File Contents
+### kind: ConfigMap
+Contains a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) which provides pods the 2 configuration files required to launch a Data Prepper instance: _pipelines.yaml_ and _data-prepper.yaml_.
+
+#### pipelines.yaml
+This file contains the standard Trace Analytics usecase pipeline configuration. A few adjustments are needed by the user:
+1. Replace the 2 `stdout` sinks with Elasticsearch sinks to actually send data to Elasticsearch.
+2. Optionally enable TLS by providing the necessary key files for the `otel_trace_source` and `peer_forwarder` plugins.
+
+#### data-prepper.yaml
```

Comment:
Shall we rename it to `data-prepper-config.yaml` or some other name? At first look, I confused it with `deployment-template/k8s/data-prepper.yaml`.

Reply:
Will rename the k8s file